src/AbstractGithubObject.php

```diff
@@ -103,6 +103,11 @@ public function __construct(Registry $options = null, BaseHttp $client = null)
 			$this->options['timeout'] = 120;
 		}
 
+		if ($this->options->get('gh.token', false))
```

Comment:
The previous path only set the Authorization if it was not set... now its being set irrespective of that.